products/opensuse/main.pm

```diff
@@ -315,21 +315,6 @@ elsif (get_var('SECURITY_TEST')) {
     prepare_target();
     load_security_tests;
 }
-elsif (get_var('VIRT_AUTOTEST')) {
```

Comment:
If this input parameter is removed, is there any impact for other jobs which was reference to this test section?

Reply:
products/opensuse/main.pm is for openSUSE tests in O3 only, product/sle/main.pm for OSD is not touched, so our SLE tests are safe.